packages/nextjs/utils/scaffold-eth/client.ts

```diff
@@ -0,0 +1,12 @@
+import { createTestClient, publicActions, walletActions, webSocket } from "viem";
+import { hardhat } from "wagmi/chains";
+
+const transport = webSocket("ws://127.0.0.1:8545");
+
+export const extendedClient = createTestClient({
```

Comment:
We can rename it to maybe `testClient`  also, maybe instead of creating new file we could collocate it inside `useFetchBlock` itself since its used only there and no other place

We can always abstract it in other file if its used at other places later 🙌